decko/features/pointer_inputs.feature

```diff
@@ -67,6 +67,23 @@ Feature: Pointer Inputs
     And I go to card "Joe User+friends"
     Then I should not see "Joe Camel"
 
+  Scenario: Creating a card with filtered list input
+    Given I create Phrase card "friends+*right+*input" with content "filtered list"
+    When I go to card "Joe User+friends"
+    And I click on "Add Item"
+    Then I should see "Select Item"
+    When I click on "Add filter"
+    And I click on "Keyword"
+    And I wait for ajax response
+    And I fill in "filter[name]" with "Joe"
```

Comment:
I know it started to do it like that in wikirate features but maybe it's time to introduce a filter step for that.